firstapp/app/src/androidTest/java/com/example/firstapp/ShellUtility.java

```diff
@@ -214,7 +216,7 @@ public long launchApp(String pkg) throws InterruptedException, IOException {
 
         //startActivity imposes a 5 second cool-down after the home button is pressed, so we wait
         //out that cool-down before grabbing the time and launching
-        //sleep(5000);
+        sleep(6000);
```

Comment:
Why the reintroduction of a sleep?

It should avoid pressing home and use shell commands as much as possible.  With that said, if it is working, then I wouldn't touch it now and file a GitHub `Issue` instead.

Reply:
For starting an app, I was getting the same issue where it would slide in from the right, which is probably not the desired visual for that. (pressing home imposes that 5 second cooldown). It also seems like crash rate might be higher if we force-stop and then immediately re-open the app.